app/views/shared/_document_overview_list.html.erb

```diff
@@ -67,6 +70,9 @@
             <li>
               <%= document.upload.filename %>
             </li>
+            <% if documents.length <= 1 && current_intake.filing_joint == "yes" %>
+              <p class="text--red-bold"><%= t("views.documents.overview.spouse_doc_needed") %></p>
```

Comment:
This <p> will end up a direct child of the <ul> which is technically not allowed, it should probably be an `<li><p>` or `<li><em>`